projects/plugins/social/src/js/components/header/index.js

```diff
@@ -19,24 +19,26 @@ import styles from './styles.module.scss';
 
 const Header = () => {
 	const {
+		connectionsAdminUrl,
 		hasConnections,
+		hasPaidPlan,
 		isModuleEnabled,
-		connectionsAdminUrl,
-		sharesCount,
-		postsCount,
 		isShareLimitEnabled,
-		hasPaidPlan,
+		newPostUrl,
+		postsCount,
+		sharesCount,
 		siteSuffix,
 	} = useSelect( select => {
 		const store = select( STORE_ID );
 		return {
+			connectionsAdminUrl: store.getConnectionsAdminUrl(),
 			hasConnections: store.hasConnections(),
+			hasPaidPlan: select( STORE_ID ).hasPaidPlan(),
 			isModuleEnabled: store.isModuleEnabled(),
-			connectionsAdminUrl: store.getConnectionsAdminUrl(),
-			sharesCount: select( STORE_ID ).getSharesCount(),
-			postsCount: select( STORE_ID ).getPostsCount(),
 			isShareLimitEnabled: select( STORE_ID ).isShareLimitEnabled(),
-			hasPaidPlan: select( STORE_ID ).hasPaidPlan(),
+			newPostUrl: `${ store.getAdminUrl() }post-new.php`,
```

Comment:
The only real change here is this one, but I felt like ordering that list alphabetically made it easier to parse, so I took that opportunity to change that.

Reply:
I don't know if that's worth it. That may just be a pet peeve of mine :)